package.json

```diff
@@ -22,11 +22,11 @@
     "node": ">=4"
   },
   "scripts": {
-    "test": "mocha --check-leaks -R spec",
+    "test": "mocha --check-leaks -R spec test/*-tests.js",
     "posttest": "npm run lint",
-    "test:coverage": "istanbul cover _mocha -- --check-leaks test/*-tests.js",
+    "test:coverage": "istanbul cover --print none --report none --include-pid _mocha -- --check-leaks test/*-tests.js && istanbul report text-summary lcov && rm -f test/coverage/coverage-*.json",
```

Comment:
I could move it into the `posttest:coverage` step. It's not part of a separate `clean` script because, if `test:coverage` is run multiple times without the `coverage-$PID.json` files being cleaned out, there's no way for Istanbul to distinguish the results of the previous runs from the results of the current run, and it will merge them into a single report.